src/mon/MonCommands.h

```diff
@@ -701,6 +701,18 @@ COMMAND("osd set-nearfull-ratio " \
 	"name=ratio,type=CephFloat,range=0.0|1.0", \
 	"set usage ratio at which OSDs are marked near-full",
 	"osd", "rw", "cli,rest")
+COMMAND("osd get-full-ratio " \
+	"name=ratio,type=CephFloat,range=0.0|1.0", \
+	"get usage ratio at which OSDs are marked full",
+	"osd", "rw", "cli,rest")
```

Comment:
I don't like these commands, as they pollute the command namespace. I would prefer these to be under a specific category:

```
osd get full-ratio
osd get backfill-full-ratio
osd get near-full-ratio
```
Also, all these three commands are read-only, not read-write.